### DISCUSSION TITLE
using old prisma version with openssl 1.1 on OS with openssl3

### DISCUSSION THREAD

**Discussion by zodman** [prisma/prisma#14010] [Q&A] [answered]
Hi guys I'm using the prisma 3.1.1 on ubuntu 22.04 what does it means, it's not compatible because ubuntu uses the latest openssl3.

then I have to download the openssl1.1 and install/compile it manually on /usr/local/

Then when I was working with Prisma I have to do  ln -s /usr/local/bin/openssl /usr/bin/ when Im don't ever the symbolic link

for it to get it working, is there a way to implement where prisma finds the openssl for this situation? for example:

`OPENSSL_BIN=/usr/local/bin/openssl1.1 npx prisma generate`

yes, we can implement docker but for our case, it's too complex...

**Comment by janpio** [comment, marked as answer]
Right now that is definitely not possible. But you could open a feature request for this, so we can evaluate if this would be a viable addition.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
